src/main/java/nextstep/security/authorization/AuthorityAuthorizationManager.java

```diff
@@ -0,0 +1,34 @@
+package nextstep.security.authorization;
+
+import jakarta.servlet.http.HttpServletRequest;
+import nextstep.security.authentication.Authentication;
+
+import java.util.Collections;
+import java.util.HashSet;
+import java.util.Objects;
+import java.util.Set;
+
+public class AuthorityAuthorizationManager implements AuthorizationManager<HttpServletRequest> {
+    private final Set<String> defined_authorities = new HashSet<>();
```

Comment:
```suggestion
    private final Set<String> definedAuthorities = new HashSet<>();
```

컨벤션에 맞게 카멜 케이스로 변경해보면 어떨까요?